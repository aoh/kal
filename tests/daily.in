# flags: -t 0 -n 14
every Monday: foo
