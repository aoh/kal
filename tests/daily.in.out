5.1.1970, Monday, week 2
 - foo

12.1.1970, Monday, week 3
 - foo

every Monday: foo
